modules/prelude-erc.el

```diff
@@ -129,6 +135,9 @@ that can occur between two notifications.  The default is
   "Connect to IRC?"
   (interactive)
   (when (y-or-n-p "Do you want to start IRC? ")
+    (if prelude-new-irc-persp
+        (progn (persp-new "IRC")
```

Comment:
I'm guessing here you should also check if perspective is loaded otherwise this would blow up.